houghtool/__init__.py

```diff
@@ -609,25 +615,30 @@ def intersection(line1, line2, tolerance=1e-6, subpixel=False):
 
 
 def draw_lines(img, lines, color=None, thickness=1):
+    num_lines_drawn = 0
     h, w = img.shape[:2]
     color = color or (0, 255, 0) if len(img.shape) == 3 else 255
     _linetype = linetype(lines[0])
     if _linetype == RHOTHETA:
         lines = [convert(line, bbox=[0, 0, w, h]) for line in lines]
     for line in lines:
         for x1, y1, x2, y2 in line:
-            if ERRORVAL in [x1, y1, x2, y2]:
-                print("Line {} has np.nan vals; not drawing.".format(line))
+            # if ERRORVAL in [x1, y1, x2, y2]:
+            if True in np.isnan([x1, y1, x2, y2]):
```

Comment:
I'm not actually sure this is any different, unless someone accidentally overwrites `ERRORVAL`

Reply:
This change was needed because np.nan is failing equality checks.
e.g. check this snippet
a = [np.nan] * 4
a[0]
Out[5]: nan
a[0] == np.nan
Out[6]: False

Reply:
Right, IEEE rules are that nans do not compare to any other value, even other nan values, which is why numpy and math have `isnan` functions. That's different than `in`, though:

```python
>>> np.nan in [1, 2, np.nan]
True
```

Reply:
In my case the "in" command also doesn't work (example snippet below):
>>line[0]
array([nan, nan, nan, nan])
>>x1, y1, x2, y2 = line[0]
>>ERRORVAL
nan
>>ERRORVAL in [x1, y1, x2, y2]
False

Reply:
however this works, probably because its the same object
```
>>ERRORVAL in [x1, y1, x2, np.nan]
True
```